lib/account_deleter.rb

```diff
@@ -69,10 +68,6 @@ def delete_standard_person_associations
     end
   end
 
-  def disconnect_contacts
```

Comment:
Don't you mind me reverting the change with `disconnect_contacts` in the account deleter?